src/main/java/org/aksw/simba/lemming/metrics/single/nodetriangles/ayz/ListingAyzMetric.java

```diff
@@ -20,8 +20,9 @@
  * @see <a href=
  *      "https://www.researchgate.net/publication/225621879_Finding_and_Counting_Given_Length_Cycles">https://www.researchgate.net/publication/225621879_Finding_and_Counting_Given_Length_Cycles</a>).
  *
+ * @see <a>https://i11www.iti.kit.edu/extra/publications/sw-fclt-05_t.pdf</a>
+ *
  * @author Tanja Tornede
```

Comment:
You can add yourself in an additional author line if you want :wink:

Reply:
Done and Thanks. :)